www/docs/externals.md

```diff
@@ -58,9 +66,73 @@ Encapsulating external reactive dependencies as eager collections, that complex
 ## Non-Skip services
 
 Of course, unless your application is built from the ground up using the Skip framework, it is likely that your application depends on some non-reactive external system: REST APIs, databases, external HTTP endpoints, and the like.
-These systems operate on a pull-based request/response paradigm, so some work is required to adapt them to Skip's eager push-based paradigm.
+
```

Comment:
I guess that right here we could add a sentence stating that the following pre-built integrations can be reimplemented with whatever customization is needed for a given use case; nothing beyond the exposed skip framework is used in the implementations.